webservices/schemas.py

```diff
@@ -213,6 +213,16 @@ class CommitteeSearchListSchema(ApiSchema):
 register_schema(CommitteeReportsSchema)
 register_schema(CommitteeReportsPageSchema)
 
+make_ie_reports_schema = functools.partial(
```

Comment:
We're only making one schema for `CommitteeReportsIEOnly`, so the functional partial doesn't buy us anything here. Also, since this model doesn't have `committee` or `report_key` fields, we don't need to `exclude` them:

```python
CommitteeReportsIEOnlySchema = make_schema(
    models.CommitteeReportsIEOnly,
    fields={
        'pdf_url': ma.fields.Str(),
    },
    options={'exclude': ('idx', )},
)
```